packages/components/cascader/__tests__/cascader.test.tsx

```diff
@@ -428,15 +429,18 @@ describe('Cascader.vue', () => {
   })
 
   test('should be able to trigger togglePopperVisible outside the component', async () => {
+    let cascader: InstanceType<typeof ElCascader>
     const clickFn = () => {
-      wrapper.findComponent(Cascader).vm.togglePopperVisible()
+      cascader.togglePopperVisible()
     }
     const wrapper = _mount(() => (
       <div>
         <Cascader options={OPTIONS} />
         <button onClick={clickFn} />
       </div>
     ))
+
+    cascader = wrapper.findComponent(Cascader).vm
```

Comment:
The effect should be the same before and after modification here, I don’t understand the difference. 🤔